app/src/main/java/dadeindustries/game/gc/model/FactionArtifacts/Unit.java

```diff
@@ -54,9 +38,73 @@ public String getShipName() {
 		return unitName;
 	}
 
+    public void setSector(Sector newlocation) {
+        currentLocation = newlocation;
+    }
+
 	private enum Order {
 		MOVE, SCUTTLE, COLONISE, BUILD_STARBASE, ATTACK_SYSTEM
 //		,RAID
 	}
 
+    private ArrayDeque<Cood> course;
+
+    public void setCourse(int destX, int destY) {
+
+        Log.wtf("Plotting", "...");
```

Comment:
What is this logging level?

Reply:
I wanted to use a different log level from E (error) because of the amount of existing noise there is on that level. There are a lot of background services erroring all over the place. I don't mind which one we decide to stick to, though. By the way, WTF stands for "What a Terrible Failure" apparently...